src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/camunda/payment/PaymentsCallbackHandler.java

```diff
@@ -85,6 +86,15 @@ private CallbackResponse makePbaPaymentBackwardsCompatible(CallbackParams callba
             .build();
     }
 
+    private CaseData updateWithDuplicatePaymentError(CaseData caseData, FeignException e) {
+        return caseData.toBuilder()
+            .paymentDetails(PaymentDetails.builder()
+                                .status(FAILED)
+                                .errorMessage(e.getMessage())
```

Comment:
What does `e.getMessage()` return here? I suspect we should use `e.contentUTF8()` to get correct error message.